### PR TITLE
Test: Feedbacks

### PR DESCRIPTION
**Pull request by arnonrdp**
## Description
Created 3 tests for feedbacks

### CODE REVIEW

**Review by hermannleboss — COMMENTED** [review]
Can you make all these test in one test, every test should be able to run indepently. 
So to be able to delete you need to create it. So in one test you can do it

File: test/cypress/e2e/feedbacks.cy.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+  it('should send a feedback message to the admin', () => {
+    cy.visit('/profile')
+    cy.get('[data-test="feedback-button"]').click()
+
+    cy.get('[data-test="subject-input"]').type('Test subject')
+    cy.get('[data-test="message-input"]').type('Test message')
+    cy.get('[data-test="submit-button"]').click()
+    cy.wait(1000)
+
```
[the remark this answers is not in the document]
Comment (arnonrdp) [reply test/cypress/e2e/feedbacks.cy.js]: Removed. Thanks.

**Comment by arnonrdp** [comment]
> Can you make all these test in one test, every test should be able to run indepently. So to be able to delete you need to create it. So in one test you can do it

Yes, I know, but by making them separate, they are more organized, explanatory and we have 3 asserts instead of just 1.

**Comment by hermannleboss** [comment]
@arnonrdp do you have error due to timeout ?

**Comment by hermannleboss** [comment]
> Yes, I know, but by making them separate, they are more organized, explanatory and we have 3 asserts instead of just 1.

I'm planning to set up retries, who will run the test when they fail, but if the test is not independent, he can fail due to dependencies.

**Comment by arnonrdp** [comment]
> I'm planning to set up retries, who will run the test when they fail, but if the test is not independent, he can fail due to dependencies.

Oh okay. It's just that you hadn't told me about those plans. But okay then.

**Pull request merged**